.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
While having this in the .gitignore isn't particularly bad, developer specific ignores should really be in your global gitignores.  Please see the following article for details:  https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
I disagree, some projects prefer sharing ide settings, others do not, not really a global setting. But I can drop it if you wish

Reply:
Done